src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -4346,7 +4362,26 @@ public override bool PreVisit(CallNode node)
                 _currentScope = scopeNew.Parent;
                 PostVisit(node.Args);
 
-                // Typecheck the invocation.
+                if (maybeFunc.HasIdentifiers && _features.HasFlag(Features.SupportColumnNamesAsIdentifiers))
+                {
+                    var i = 0;
+
+                    foreach (var arg in args)
+                    {
+                        i++;
+
+                        if (arg is FirstNameNode firstNameNode && maybeFunc.IsIdentifierParam(i))
+                        {
+                            var displayName = GetLogicalNodeNameAndUpdateDisplayNames(argTypes[0], firstNameNode.Ident, out _);
+
+                            if (argTypes[0].TryGetType(displayName, out var fieldType))
+                            {
+                                var fileNameInfo = FirstNameInfo.Create(firstNameNode, new NameLookupInfo(BindKind.NamedValue, fieldType, DPath.Root, 0, displayName: new DName(displayName)));
```

Comment:
>BindKind.NamedValue [](http://example.com/codeflow?start=106&length=19)

How did you decide to use this value? (Does it actually matter)
There are some other interesting candidates, like LambdaField

Reply:
Code removed, see other comments